.vscode/launch.json

```diff
@@ -0,0 +1,40 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
Should add .vscode/ to the .gitignore probably

Reply:
Frontend ignores it for what it's worth.